src/CMakeLists.txt

```diff
@@ -91,3 +93,10 @@ if(CLANG_FORMAT)
   add_custom_target(CL_FORMAT_CSRCS COMMAND ${CLANG_FORMAT_EXEC} -i -style=file ${ALL_CSRCS})
   add_dependencies(torch_xpu_ops CL_FORMAT_CSRCS)
 endif()
+
+find_package(ONEMKL)
+if(NOT ONEMKL_XPU_FOUND)
+  message(FATAL_ERROR "Can NOT find ONEMKL cmake helpers module!")
+endif()
```

Comment:
Move to top level of CMAKE, since the variables. defined in FindONEMKL are global. You can add a file `cmake/MKL.cmake`, and move the code piece there.

Reply:
Done. BTW, the prefix `ONEMKL_XPU` has been changed to `ONEMKL`.